### PR TITLE
#1767 Add validateApi unit test

### PR DESCRIPTION
**Pull request by DavidMi-1**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1767)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/api/__tests__/validateApi.test.js [hunk 1]
```diff
@@ -14,26 +14,28 @@
...
+describe('validateNode()', () => {
   const params = {
-    hostname: 'abc',
-    port: 22,
-    user: 'usr',
-    password: 'ps',
+    hostname: generate.name(),
+    port: generate.port(),
+    user: generate.userName(),
+    password: generate.name(),
```
Comment (konekoya) [inline ohara-manager/client/src/api/__tests__/validateApi.test.js]: Let's create a `generate.password()` here. Ref: https://github.com/Marak/faker.js

Comment (DavidMi-1) [reply ohara-manager/client/src/api/__tests__/validateApi.test.js]: ok, got it

**Comment by DavidMi-1** [comment]
retry manager

**Comment by konekoya** [comment]
@DavidMi-1 build failed. Could you try running `yarn prepush` or simply `yarn foramt` in your local machine?

**Comment by DavidMi-1** [comment]
> @DavidMi-1 build failed. Could you try running `yarn prepush` or simply `yarn foramt` in your local machine?

ok , I run them at local again

**Comment by konekoya** [comment]
Great, everything is passing. Thanks for the PR @DavidMi-1

**Pull request merged**